compiler/rustc_errors/src/emitter.rs

```diff
@@ -417,7 +417,7 @@ pub trait Emitter {
                 if !redundant_span || always_backtrace {
                     let msg: Cow<'static, _> = match trace.kind {
                         ExpnKind::Macro(MacroKind::Attr, _) => {
-                            "this procedural macro expansion".into()
+                            "this macro attribute expansion".into()
```

Comment:
I'd say for consistency this should either state `this attribute macro expansion` or for derives below to be changed to `this macro derive expansion`